### PR TITLE
Fixed the issue #171

### PR DESCRIPTION
**Pull request by ayush-sleeping**
### Please make sure you check all the items.

- [x] The numbering of the company that you added starts with ```1```.
- [x] You have increased the ```Total Companies Added``` count.

Fixed the issue -> Career page of Nykaa , EXL Healthcare and Intellika #171

### CODE REVIEW

**Comment by ayush-sleeping** [comment]
Hey  @Kaustubh-Natuskar ,  <br> 
Review my changes and if all things are fine kindly merge it .

File: README.md [hunk 1]
```diff
@@ -257,6 +259,7 @@ PLEASE DON'T CHANGE THE NUMBERING. KEEP EVERYTHING STARTING WITH 1.
 1.   [nurture.farm](https://nurture.farm/join-us/)  
 1.   [Nutanix](https://www.nutanix.com/company/careers)   
 1.   [Nvidia](https://www.nvidia.com/en-in/about-nvidia/careers/)  
+1.   [Nykaa](https://www.nykaa.com/careers?gclid=CjwKCAiAv_KMBhAzEiwAs-rX1AaeJPV4qukBH6DJnWPnJfUpAwGN9pMmMU6hYHyEfUELj3RNCKRkkBoCVHAQAvD_BwE&ef_id=CjwKCAiAv_KMBhAzEiwAs-rX1AaeJPV4qukBH6DJnWPnJfUpAwGN9pMmMU6hYHyEfUELj3RNCKRkkBoCVHAQAvD_BwE:G:s&s_kwcid=AL!599!3!560340772706!!!g!!&utm_source=GooglePaid&utm_medium=search&utm_campaign=adwords_search_dsa_full_site)
```
Comment (ayush-sleeping) [inline README.md:262]: here is Nykaa career page link

**Comment by Kaustubh-Natuskar** [comment]
Why nyka's link so lengthy ?

File: README.md [hunk 2]
```diff
@@ -192,7 +193,8 @@ PLEASE DON'T CHANGE THE NUMBERING. KEEP EVERYTHING STARTING WITH 1.
 1.   [Informatica](https://informatica.gr8people.com/jobs)    
 1.   [Inmobi](https://www.inmobi.com/company/careers/)  
 1.   [Innovaccer](https://innovaccer.com/careers/)  
-1.   [Intel](https://www.intel.com/content/www/us/en/jobs/jobs-at-intel.html)  
+1.   [Intel](https://www.intel.com/content/www/us/en/jobs/jobs-at-intel.html)
+1.   [Intellika](https://www.intellika.in/careers/)
```
Comment (ayush-sleeping) [inline README.md:197]: here is Intellika career page link

File: README.md [hunk 3]
```diff
@@ -133,7 +133,8 @@ PLEASE DON'T CHANGE THE NUMBERING. KEEP EVERYTHING STARTING WITH 1.
 ## E  
 1.   [Enfussion](https://enfusion.com/careers/)
 1.   [Envestnet Yoodlee](https://www.yodlee.com/oceania/india-careers) 
-1.   [Eurofins](https://careers.eurofins.com/in/)  
+1.   [Eurofins](https://careers.eurofins.com/in/)
+1.   [EXL Healthcare](https://exlservice.com/careers)
```
Comment (ayush-sleeping) [inline README.md:137]: here is EXL career page link

**Comment by ayush-sleeping** [comment]
> Why nyka's link so lengthy ?

I don't know .... it is on their official page

**Comment by ayush-sleeping** [comment]
Hey @Kaustubh-Natuskar , <br>
What happened , You haven't merge this " PR " yet .

**Comment by ayush-sleeping** [comment]
Hey @Kaustubh-Natuskar ,

What happened , You haven't merge this " PR " yet .

**Pull request merged**